R/rABS.R

```diff
@@ -0,0 +1,94 @@
+#' ABS simulation
+#'
+#' This function is for simulating the Auto-correlated Bayesian Sampler \insertCite{zhuAutocorrelatedBayesian2023}{samplr}.
+#'
+#' @param dec_bdry The decision boundary that separates the posterior hypothesis distribution.
+#' @param discrim The stimuli discriminability.
+#' @param delta The threshold of the relative difference. It should be an integer.
+#' @param nd_time The non-decision time.
+#' @param s_nd_time The range of the non-decision time.
+#' @param lambda The rate parameter of the gamma distribution for decision time.
+#' @param trial_fdbk The feedback of each trial. It should be a character only consist of 0 (below the decision boundary) and/or 1 (above the decision boundary).
+#' @param nChains The number of chains of the sampler. It should be an integer.
+#' @param width The proposal width of the sampler.
+#' @param distr_name The type of the posterior hypothesis distribution.
+#' @param mc3_iterations The number of iterations for each chunk. See details for more information.
+#' 
+#' @return A data frame with ten columns:
+#' \enumerate{
+#'  \item{trial: the index of trials;}
+#'  \item{samples: the samples of ABS sampler for the trial;}
+#'  \item{support: the relative difference between the numbers of supporting samples for each response;}
+#'  \item{length: the length of the sequence;}
+#'  \item{response: the response predicted by ABS. 0 represents the response coded as below the decision boundary and 1 represent the opposite;}
+#'  \item{feedback: the feedback or the true stimuli of the experiment. 0 represents the response coded as below the decision boundary and 1 represent the opposite;}
+#'  \item{accuracy: whether the response is the same as the feedback. 0 represents error, and 1 represents correct;}
+#'  \item{nd_time: the non-decision time for this trial;}
+#'  \item{rt: the response time, including both the non-decision and the decision time;}
+#'  \item{rept: whether the next trial repeats the same response as the current trial;}
+#' }
+#' 
+#' @details
```

Comment:
Please add details above, before the description of parameters (https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html#the-introduction)